tensorboard/backend/application.py

```diff
@@ -85,6 +86,7 @@ def standard_tensorboard_wsgi(
     reload_interval: The interval at which the backend reloads more data in
         seconds.
     plugins: A list of constructor functions for TBPlugin subclasses.
+    base_url: A prefix of the path when app isn't served from root.
```

Comment:
Place `base_url` last (because it is the last of the arguments). As noted above, consider renaming to `path_prefix`.